### PR TITLE
Column filter functionality and render on browser window

### PR DESCRIPTION
**Pull request by zbuzzeo**

### CODE REVIEW

File: migrations/20190212102119_cards.js [hunk 1]
```diff
@@ -2,9 +2,10 @@
 exports.up = function(knex, Promise) {
   return knex.schema.createTable('cards', (table) => {
     table.increments().primary();
+    table.string('title').notNullable();
```
Comment (taesup) [inline migrations/20190212102119_cards.js:5]: You shouldn't modify migration files. You should make a new migration instead.

File: src/components/Column/Column.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Card from '../Card';
+import './Column.scss';
+
+const Column = (props) => {
+  // for every card Object in the array, we want to inject its properties into its corresponding component.
+  console.log(props);
```
Comment (taesup) [inline src/components/Column/Column.jsx]: no console.logs

**Review by taesup — CHANGES_REQUESTED** [review]
So from here now out, make sure you keep making the changes, merging in the PRs, and keep doing work in a new branch that is targetted to PR against development. This keeps the number of files to review down.

File: migrations/20190212102119_cards.js (hunk 1 above)
Comment (zbuzzeo) [reply migrations/20190212102119_cards.js:5]: Made the change in a new commit.

File: src/components/Column/Column.jsx (hunk 2 above)
Comment (zbuzzeo) [reply src/components/Column/Column.jsx]: Made the change in a new commit.

File: migrations/20190212102119_cards.js (hunk 1 above)
Comment (jaywon) [reply migrations/20190212102119_cards.js:5]: If you made changes in a new commit, that should mean that these changes are removed from this commit and don't show up in a diff

Comment (jaywon) [reply migrations/20190212102119_cards.js:5]: I see in your other commit you did a `createTable`, what you want to do is leave this commit as is for the `createTable` and your next migration would be to modify the table columns that you want to change.

File: server/routes/board.js [hunk 3]
```diff
@@ -4,8 +4,23 @@ const express = require('express');
 
 const router = express.Router();
 
+const books = []; // use bookshelf to grab card data from database
+
 router.get('/', (req, res) => {
-  // show kanban board
+  console.log('fucking hitty');
```
Comment (jaywon) [inline server/routes/board.js]: Nope. 🔥

Comment (jaywon) [reply server/routes/board.js]: (Remember this is a public record of your work and professionalism), offline is fine, in code can be used against you

File: server/server.js [hunk 4]
```diff
@@ -4,20 +4,23 @@ const bodyParser = require('body-parser');
...
 
-app.use(bodyParser.urlencoded({ extended: false }));
+if (!REDIS_HOST_PORT) {
+  throw new Error('REDIS_HOST_PORT not set');
+}
+
 app.use(bodyParser.json());
 app.use(session({
-  store : new redis({ url : 'redis://redis-server:6379', logErrors : true }),
+  store : new redis({ url : `redis://redis-server:${REDIS_HOST_PORT}`, logErrors : true }),
```
Comment (jaywon) [inline server/server.js]: Both the REDIS_HOST and REDIS_HOST_PORT should be environment variables. In this case your solution is fixed to a single host address.

File: src/containers/App/App.js [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+      headerTitle : `KANBAN`,
+    }
+  }
+
+  render() {
+    return (
+      <div className="App">
+        <Header title={ this.state.headerTitle } />
+        {/* the Board component should have a cards prop referencing the array of cards that we get from the reducer. */}
+        <Board cards={ cardReducer } />
```
Comment (jaywon) [inline src/containers/App/App.js]: Don't think you need to pass in the reducer to your component in this way. That's what the redux `connect` method will do on your `Board` component.

**Review by jaywon — CHANGES_REQUESTED** [review]
Couple more small changes plz 🙏

**Review by jaywon — APPROVED** [review]
LGTM ✅

**Pull request merged**